### PR TITLE
[netflow] Add NDM NetFlow static enrichments

### PR DESCRIPTION
**Pull request by AlexandreYang**
Requires https://github.com/DataDog/datadog-agent/pull/10976

### What does this PR do?

Add NDM NetFlow static enrichments 
- Direction
- Network Mask
- IP Protocol
- Ether Type
- Mac Address
- TCP Flags

### Motivation

Enrichment needed for NetFlow feature

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by alai97 — APPROVED** [review]

**Review by gmarabout — APPROVED** [review]

**Review by pducolin — APPROVED** [review]
Comments are not blocking

File: pkg/netflow/enrichment/ether_type.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	0x86DD: "IPv6",
+}
+
+// MapEtherType map Ether Type number to human-readable Ether Type name
+// https://www.iana.org/assignments/ieee-802-numbers/ieee-802-numbers.xhtml
+func MapEtherType(etherTypeNumber uint32) string {
+	protoStr, ok := etherTypeMap[etherTypeNumber]
+	if !ok {
+		return ""
+	}
```
[the remark this answers is not in the document]
Comment (AlexandreYang) [reply pkg/netflow/enrichment/ether_type.go:20]: That seems an alternative, but there a couples of reasons I would prefer keep the `ok` check.

- I'm not sure if the map returned value behaviour is documented when there the entry is not found
- I find it more readable to explicitly return an empty string than return the returned value (the user needs to check map documentation to verify maps returns a zero/empty value when no entry is found)

[the remark this answers is not in the document]
Comment (pducolin) [reply pkg/netflow/enrichment/ether_type.go:20]: I agree on making it more readable for non golang experts. Map's behavior on non existing keys is [documented here](https://go.dev/blog/maps)

> If the requested key doesn’t exist, we get the value type’s zero value.

Zero value for strings is `""` (source https://go.dev/tour/basics/12)

[the remark this answers is not in the document]
Comment (AlexandreYang) [reply pkg/netflow/enrichment/ether_type.go:20]: Thanks for looking at the doc for this, good to know that this is a documented behaviour for maps ☝️

**Pull request merged**